src/wifihandler.cpp

```diff
@@ -72,6 +80,30 @@ void onConnected() {
     onWiFiConnected();
 }
 
+
+void wifiNoCredsloop() {
+    if (WIFI_NOCREDS_CONNECT == WIFI_SMARTCONFIG) {
```

Comment:
Since `WIFI_NOCREDS_CONNECT ` is a define, maybe convert this if operator to `#if/elif/endif` directive so it compiles only necessary code?
```c
#if WIFI_NOCREDS_CONNECT == WIFI_SMARTCONFIG
...
#elif WIFI_NOCREDS_CONNECT == WIFI_HOTSPOT
...
#endif
```

Reply:
i agree, i making the fix